### PR TITLE
Document package command getprojectservices with OpenAPI

### PR DESCRIPTION
**Pull request by eduardoj**
### For reviewers

Access to the documented enpoint in the review-app through [this link](https://obs-reviewlab.opensuse.org/eduardoj-apidocs_newsource_package_cmd_getprojectservices/apidocs-new/index.html#/Sources%20-%20Packages/post_source__project_name___package_name__cmd_getprojectservices).

### CODE REVIEW

File: src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_getprojectservices.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+post:
+  summary:  List all services related to a package.
+  description: |
+    List all services defined in the project spaces defined for this package.
```
Comment (krauselukas) [inline src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_getprojectservices.yaml:4]:
```suggestion
    List all services defined in the project space for this package.
```

Comment (krauselukas) [reply src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_getprojectservices.yaml:4]: or maybe this describes it a little better? https://github.com/openSUSE/open-build-service/blob/fa5742de6c10e9ac12bc2ef1e3603f0229faa3c9/src/backend/bs_srcserver#L904

Comment (eduardoj) [reply src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_getprojectservices.yaml:4]: I took the line of the old documentation, which states: "getprojectservices: list all service defined in project spaces defined for this package."

I also found tests for this endpoint: https://github.com/openSUSE/open-build-service/blob/8d1d9658185e39400b47718cb8c3bfdb6e528e08/src/api/test/functional/source_services_test.rb#L36-L57

In this case I prefer to follow the initial idea of "projects spaces". Is there any reason to switch to "project space" I'm missing?

Comment (eduardoj) [reply src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_getprojectservices.yaml:4]: Anyway, it makes sense to remove `defined`.

Comment (eduardoj) [reply src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_getprojectservices.yaml]: I removed the last `defined`.

**Review by krauselukas — APPROVED** [review]

**Pull request merged**